packages/core/src/___internal-do-not-use-will-break-in-patch/admin-ui/pages/ListPage/index.tsx

```diff
@@ -254,7 +254,7 @@ const ListPage = ({ listKey }: ListPageProps) => {
               <FilterAdd listKey={listKey} filterableFields={filterableFields} />
             ) : null}
             {filters.filters.length ? <FilterList filters={filters.filters} list={list} /> : null}
-            {Boolean(Object.keys(query).length) && (
+            {Boolean(filters.filters.length || sort || query.fields) && (
```

Comment:
`sort` will be truthy when there is an initial sort so we'll want to only show the button when there's a custom sort
```suggestion
            {Boolean(filters.filters.length || query.sortBy || query.fields) && (
```